src/core/p5.Renderer2D.js

```diff
@@ -389,7 +388,7 @@ p5.Renderer2D.prototype._acuteArcToBezier =
 p5.Renderer2D.prototype.arc =
   function(x, y, w, h, start, stop, mode) {
   var ctx = this.drawingContext;
-  var vals = canvas.arcModeAdjust(x, y, w, h, this._ellipseMode);
+  var vals = {x: x, y: y, w: w, h: h};
```

Comment:
Maybe we can lose vals here? I know it requires updating more lines of code, but I don't think anything is gained by putting these parameters into an object now that the core p5 module does the adjustment of the coordinates.

Reply:
Fixed as part of removing arcModeAdjust as its own function